src/ui/components/TransferGrain.js

```diff
@@ -18,90 +17,151 @@ export type Props = {|
   +setLedger: (Ledger) => void,
 |};
 
+const useStyles = makeStyles({
```

Comment:
If you need to access values from the theme, you can do this:
```suggestion
const useStyles = makeStyles((theme) => ({
  root: {
     background: theme.palette.background,
```

Just an example to illustrate, you probably dont even need to set the BG if you just use the provided components from MUI.